miniscope_io/formats/stream.py

```diff
@@ -5,10 +5,10 @@
 from miniscope_io.models.stream import StreamBufferHeaderFormat
 
 StreamBufferHeader = StreamBufferHeaderFormat(
-    linked_list=(0, 32),
-    frame_num=(32, 64),
-    buffer_count=(64, 96),
-    frame_buffer_count=(96, 128),
-    timestamp=(192, 224),
-    pixel_count=(224, 256),
+    linked_list=0,
+    frame_num=1,
+    buffer_count=2,
+    frame_buffer_count=3,
+    timestamp=6,
```

Comment:
i guess question leftover from before, what is in positions 4 and 5 here?

Reply:
I think they were buffer-counting and more specific timestamp stuff we didn't need for initial tests (there's actually metadata in 7-10, too).

I was thinking we could complete this later when we have an idea about the metadata we want to send, but it's an easy change anyway, so I'll add them before merging this.

Reply:
sure sure - mostly curiosity. would be good to have these in sync with whatever is coming off the FPGA/in the test data, but not totally critical at the moment